app/utils.py

```diff
@@ -15,3 +16,41 @@ def csv_reader(path_to_file):
         csvreader = csv.DictReader(csvfile)
         for row in csvreader:
             yield row
+
+
+class UserSkillMatch(object):
+    """
+    An object linking a User with matched skills.
+
+    This object exposes `user` and `questionnaires` properties.  The
+    `questionnaires` property is a list of two-tuples, in descending order of
+    number of matched questions, where the first element is the name of the
+    questionnaire and the second element is a list (in no particular order) of
+    the the matched quesiton IDs.
```

Comment:
This is really great! The only thing that would make it even more rad would be really simple doctests to give the reader some usage examples, but that's really just extra icing on the cake.